packages/loader/driver-definitions/src/storage.ts

```diff
@@ -319,4 +319,10 @@ export interface ISummaryContext {
     readonly ackHandle: string | undefined;
 
     readonly referenceSequenceNumber: number;
+
+    /**
+     * Indicates that this is the first summary for a previously created empty file.
+     * The server should use this as the first summary for the file without any corresponding summary op.
```

Comment:
Maybe 
> Indicates that this is the first summary for a previously created empty file.
Such summary always full summary (contains .protocol tree, has no parent summary, no references to prior summaries).
Storage service should commit such summary without need for client to involve ordering service (send summary op).

That said, can we use ackHandle === undefined as same hint? Do we ever have undefined parent summary today? If yes, why and what does it mean? It feels like it's redundant info.

Reply:
> That said, can we use ackHandle === undefined as same hint? 

I have made this change in #7110 (which makes this PR unnecessary).

>Do we ever have undefined parent summary today? If yes, why and what does it mean?

I don't think so, but I'm not sure. ODSP gives a 400 response if a container payload is sent with a sequence number > 0, so I think it will be obvious if this assumption is invalid.